_infographics/emisie/emisie-sr.md

```diff
@@ -17,18 +17,18 @@ data-orig:  [
 ## Ako čítať tento graf a čo znamenajú jednotlivé sektory?
 
 * Všetky hodnoty emisií uvádzame v jednotkách <glossary id="co2eq">CO<sub>2</sub>eq</glossary>.
-* Ročný objem emisií Slovenska je 43,53 mil. ton CO<sub>2</sub>eq (údaj z roku 2018). __V prepočte na obyvateľa to je 8t CO<sub>2</sub>eq/obyvateľa.__
```

Comment:
Vsetko OK, len nerozumiem, preco si tu zmazala medzeru pred tonami @LenkaSedlackova. Nemala by tam byt?

Reply:
to tam práve už bolo v origináli a zabudla som to opraviť :))) som na to pozerala a po prekliku do edit modu mi padlo to oka niečo iné, tak som sa k tomu zabudla vrátiť

Reply:
btw - už si neviem vytvoriť vlastný branch na edity?